persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/StringStoreWithManagedConnectionFunctionalTest.java

```diff
@@ -13,30 +10,10 @@
 @Test(groups = "functional", testName = "persistence.jdbc.stringbased.StringStoreWithManagedConnectionFunctionalTest")
 public class StringStoreWithManagedConnectionFunctionalTest extends AbstractStringBasedCacheStore {
 
-    private ManagedConnectionFactoryConfiguration customFactoryConfiguration;
-
-    public StringStoreWithManagedConnectionFunctionalTest() {}
-
-    @SuppressWarnings("unused")
-    public StringStoreWithManagedConnectionFunctionalTest(ManagedConnectionFactoryConfiguration customFactoryConfiguration) {
-        this.customFactoryConfiguration = customFactoryConfiguration;
-    }
-
     @Override
     protected ConnectionFactory getConnectionFactory(JdbcStringBasedStoreConfigurationBuilder storeBuilder) {
-        if(customFactoryConfiguration != null) {
-            ConnectionFactoryConfiguration connectionFactoryConfiguration = storeBuilder
-                    .dataSource()
-                    .read(customFactoryConfiguration)
-                    .create();
-            final ConnectionFactory connectionFactory = ConnectionFactory.getConnectionFactory(ManagedConnectionFactory.class);
-            connectionFactory.start(connectionFactoryConfiguration, connectionFactory.getClass().getClassLoader());
-            return connectionFactory;
-        } else {
-            SimpleConnectionFactory simpleFactory = new SimpleConnectionFactory();
-            simpleFactory.start(configureSimpleConnectionFactory(storeBuilder).create(), Thread.currentThread().getContextClassLoader());
-            return simpleFactory;
-        }
-
+        SimpleConnectionFactory simpleFactory = new SimpleConnectionFactory();
```

Comment:
Based on our previous discussions, it seems like this change is fixing your issue because we're no longer using the `ManagedConnectionFactory`

Reply:
Actually on jdg-functional-tests I'm overriding this method and testing using `ManagedConnectionFactory`

Reply:
Great, just wanted to make sure :slightly_smiling_face: